style.css

```diff
@@ -0,0 +1,111 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+  list-style: none;
+  font-family: 'Roboto', sans-serif;
+}
+
+.logo {
+  text-decoration: none;
+}
+
+.header {
+  background-image: url(./Top-circles.png);
+  background-repeat: no-repeat;
+  background-color: #191765;
+  height: 100%;
+  padding: 0 40px 40px 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.nav a {
+  color: #fff;
+}
+
+.header h1 {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 35px;
+  line-height: 72px;
+  color: #7d73fc;
+  padding: 63px 0;
+}
+
+.header h2 {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  color: #edecff;
+  padding-bottom: 40px;
+}
+
+.header p {
+  width: 327px;
+  height: 170px;
+  font-style: normal;
+  font-weight: normal;
```

Comment:
- [ ] As mentioned before, the `font-weight` is still a bit off, please consider changing it to match the design better.